src/mono/netcore/System.Private.CoreLib/src/System/Threading/TimerQueue.Browser.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It'd be nice to match existing naming convention with .Mono.cs suffix for any Mono runtime dependent code

Otherwise LGTM

Reply:
What should the name be ? Browser.Mono.cs ?

Reply:
yep, similar to e.g. https://github.com/vargaz/runtime/blob/wasm-timers/src/mono/netcore/System.Private.CoreLib/src/System/Threading/EventWaitHandle.Unix.Mono.cs

Reply:
Renamed it, but i thought that the .Mono suffix was meant to distinguish mono versions of common sources. There is no TimerQueue.Browser anywhere else.

Reply:
Then why is TimerQueue class partial if it does not have any other parts?